Dependencies/xr/Source/ARKit/XR.mm

```diff
@@ -1185,6 +1189,10 @@ void GetHitTestResultsLegacy(std::vector<HitResult>& filteredResults, xr::HitTes
         return m_impl->sessionImpl.GetPlaneByID(planeID);
     }
 
```

Comment:
Does iOS not need a stub implementation for System::Session::Frame::GetSceneObjectByID?

Reply:
yes it does